samples/tfm_integration/psa_crypto/Kconfig

```
@@ -19,6 +19,11 @@ config PSA_SHELL
 	help
 	  Enabling this option will make the 'psa' shell command available.
 
+config PSA_IMPORT_KEY
+	bool "Enable support for importing private key data"
+	help
+	  Enable support for importing private key data using PSA APIs.
```

This description is probably a bit ambiguous, and would be clearer with
something like this to contrast with randomly generating the key.

```suggestion
	  Enable support for importing a pre-generated or randomly generated
	  private key using PSA APIs and PRIVATE_KEY_STATIC or
	  PRIVATE_KEY_RANDOM.
```

---

Fixed.